app/hooks/useLazyMint.tsx

```
@@ -0,0 +1,69 @@
+  const [txnHash, setTxnHash] = useState('');
+  const [error, setError] = useState<Error>();
+  const [isProcessing, setIsProcessing] = useState(false);
+
+  const handleLazyMint = useCallback(
+    async (args: LazyMintArgs) => {
+      // TODO: get this validated
+      (Object.keys(args) as (keyof LazyMintArgs)[]).forEach((key) => {
+        if (!args[key]) {
+          // throw new Error(`${key} is required`);
+        }
+      });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)